examples/src/main/java/io/dapr/examples/workflows/README.md

```diff
@@ -652,4 +651,37 @@ Key Points:
 1. Each successful booking step adds its compensation action to an ArrayList
 2. If an error occurs, the list of compensations is reversed and executed in reverse order
 3. The workflow ensures that all resources are properly cleaned up even if the process fails
-4. Each activity simulates work with a short delay for demonstration purposes
\ No newline at end of file
+4. Each activity simulates work with a short delay for demonstration purposes
+
+
+### Suspend/Resume Pattern
+
+Workflow instances can be suspended and resumed. This example shows how to use the suspend and resume commands. 
+
+For testing the suspend and resume operations we will use the same workflow definition used by the DemoExternalEventWorkflow.
+
+Start the workflow and client using the following commands:
+
+```sh
+dapr run --app-id demoworkflowworker --resources-path ./components/workflows -- java -jar target/dapr-java-sdk-examples-exec.jar io.dapr.examples.workflows.suspendresume.DemoSuspendResumeWorker
+```
+
+```sh
+java -jar target/dapr-java-sdk-examples-exec.jar io.dapr.examples.workflows.suspendresume.DemoSuspendResumeClient
+```
+
+The worker logs:
```

Comment:
Can you add mechanical markdown so this is tested in CI - unless Im missing it?

Reply:
added... waiting for CI to validate